src/styles/blocks/calendar.scss

```
@@ -0,0 +1,89 @@
+
+.calendar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__days {
+    display: flex;
+    flex-wrap: wrap;
+    width: $calendar-size;
+    margin: $blocks-padding;
+
+    @for $day from 1 through 7 {
+      &--#{$day} {
+        box-sizing: border-box;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        background-color: rgb(76, 231, 218);
+        height: 45px;
+        width: $squre-size;
+        border: 1px solid black;
+        margin: $small-blocks--margin;
+      }
+    }
+  }
+  
+  &__month {
+    display: flex;
+    flex-wrap: wrap;
+    width: 707px;
+    margin: $blocks-padding;
+
+    
+    &-length {
+      box-sizing: border-box;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      background-color: #eee;
+      height: $squre-size;
+      width: $squre-size;
+      border: 1px solid black;
+      margin: $small-blocks--margin;
+      cursor: pointer;
+    }
+
+    $days:
+      "mon" 0,
+      "tue" $block-size,
+      "wed" 2*$block-size,
+      "thu" 3*$block-size,
+      "fri" 4*$block-size,
+      "sat" 5*$block-size,
+      "sun" 6*$block-size;
+
+    @each $name, $value in $days {
+      &--start-days--#{$name} div:nth-child(1) {
+        margin-left: $value;
+      }
+    }
+    
+
+    @for $index from 1 through 31 {
+      &-length:nth-child(#{$index})::before {
+        content: "#{$index}";
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+        transition-property: transform;
+      }
+    }
+
+    &-length:hover {
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      background-color: rosybrown;
+      transform: translate(0, -20px);
+      transition-duration: 0.5s;
```

```suggestion
      transition-duration: 0.5s;
```
якщо перенести transition-duration з hover в блок, то анімація буде плавною, не тільки при наведенні, але й при прибиранні курсору з блоку